scripts/importers/finish.sql

```diff
@@ -44,3 +49,4 @@ alter table mofo add constraint mofo_email_id_fkey foreign key (email_id) refere
 
 -- One more cleanup
 update newsletters set "format" = 'T' where "format" = 'N';
+alter table newsletters drop column source, add column source text;
```

Comment:
What was the purpose of this cleanup?

Reply:
That was the empty-string for source instead of null cleanup